draft-ietf-core-dns-over-coap.md

```diff
@@ -403,15 +405,15 @@ When a DNS error—NxDomain (RCODE = 3) for "does.not.exist" in this case—is n
 As described in {{sec:content-format}}, a DoC server uses NotImp (RCODE = 4) if it does not support an OPCODE—a DNS Update (OPCODE = 5) for "example.org" in this case.
 
     2.05 Content
-    Content-Format: application/dns-message
+    Content-Format: 553 (application/dns-message)
     Payload (human-readable):
       ;; ->>Header<<- opcode: UPDATE, status: NOTIMP, id: 0
       ;; flags: qr ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ARCOUNT: 0
 
       ;; QUERY SECTION:
       ;example.org.                 IN      AAAA
 
-When an error occurs at the CoAP layer, the DoC server SHOULD respond with
+When an error occurs at the CoAP layer, the DoC responds with
```

Comment:
s/the DoC responds/the DoC server responds

Reply:
```suggestion
When an error occurs at the CoAP layer, the DoC server responds with
```